CHANGES.md

```diff
@@ -1,6 +1,10 @@
 ## 1.5.0
 
+* Additional columns added for po_lines_locations:
+  pol_location_name and pol_location source to accomodate data structure change with adding holding_id field for po lines,
+  Added comments to all columns for po_lines_locations.sql
 * A new derived table for LDP 1.x, `instance_administrative_notes`,
```

Comment:
If you want to leave  the  updated holdings_administrative_notes.sql on comments in this PR. We need to have it mentioned in the Changes.md as well.